tools/sync-commits-with-smithy-rs/src/main.rs

```diff
@@ -0,0 +1,318 @@
+use anyhow::{bail, Context, Result};
+use git2::{Commit, Direction, IndexAddOption, ObjectType, Oid, Repository, Signature, Sort};
+use std::fs::OpenOptions;
+use std::io::{Read, Write};
+use std::path::{Path, PathBuf};
+use std::process::Command;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(name = "sync")]
+struct Opt {
+    #[structopt(long, parse(from_os_str))]
+    from_repo_at_path: PathBuf,
+    #[structopt(long, parse(from_os_str))]
+    to_repo_at_path: PathBuf,
```

Comment:
`--to`?

Or: `smithy-rs` and `aws-sdk`

Reply:
While writing the tool I've been torn between making it specific and making it more generic. I'll go with specific from now on.